edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/persistence/AddressRepository.java

```diff
@@ -0,0 +1,10 @@
+package org.dcsa.edocumentation.domain.persistence;
+
+import org.dcsa.skernel.domain.persistence.entity.Address;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface AddressRepository extends JpaRepository<Address, UUID> { }
```

Comment:
We also have this in JIT.  I think it is time to move it to Shared Kernel.

Reply:
I agree, there are several things I would like to move to shared-kernel.